src/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.cpp

```diff
@@ -181,7 +181,7 @@ Pass Orbit::getPassWithRevolutionNumber(const Integer& aRevolutionNumber, const
 
     const std::lock_guard<std::mutex> lock {this->mutex_};
 
-    const auto getClosestPass = [this](const Integer& aRevolutionNumber) -> Pass
+    const auto getClosestPass = [this](const Integer& revolutionNumber) -> Pass
```

Comment:
Why not extract this lambda and make it its own private function so its easier to unit test

Reply:
It would be a private function that would not be unit tested in either case. We're unit testing this by testing the public interface.

Reply:
We do unit test our private functions though. But anyways feel free to resolve